linetools/isgm/utils.py

```diff
@@ -362,3 +366,138 @@ def synthesize_components(components, zcomp=None, vbuff=0*u.km/u.s):
 
     # Return
     return synth_comp
+
+
+def overlapping_chunks(chunk1, chunk2):
+    """True if there is overlap between chunks
+    `chunk1` and `chunk2`. Otherwise False. Chunks are
+    assumed to represent continuous coverage, so the only
+    information that matters are the minimum and maximum
+    values of a given chunk.
+
+    Parameters
+    ----------
+    chunk1 : tuple, list, 1-d np.array, Quantity
+        A given chunk, assumed to represent a contiguous region
+        So only its minimum and maximum values matter
+        Still, chunk must be sorted.
```

Comment:
I didn't catch the 'must be sorted' on my first read.  Might see if you can make that more prominent.